src/lib/support/DefaultStorageKeyAllocator.h

```diff
@@ -197,6 +197,12 @@ class DefaultStorageKeyAllocator
         return StorageKeyName::Formatted("g/su/%x", static_cast<unsigned>(index));
     }
     static StorageKeyName SubscriptionResumptionMaxCount() { return StorageKeyName::Formatted("g/sum"); }
+
+    static StorageKeyName FabricScenesKey(chip::FabricIndex fabric) { return StorageKeyName::Formatted("f/%x/s", fabric); }
+    static StorageKeyName FabricSceneKey(chip::FabricIndex fabric, uint8_t id)
+    {
+        return StorageKeyName::Formatted("f/%x/s/%x", fabric, id);
```

Comment:
This collides with FabricSession keys, no?  It's kind of saved by the fact that this will always have only 2 hex digits and FabricSession has 16, but it would be better to just use a non-colliding prefix for scenes, so "sc" instead of "s" or something.